### DISCUSSION TITLE
Choosing one N_cells_per_location for multiple samples

### DISCUSSION THREAD

**Discussion by ThomasThaewel** [BayraktarLab/cell2location#251] [General]
Thank you for creating this wonderful tool. 
I just have a question concerning the hyper parameter n_cells_per_location.
We are working with tissue that shows high amounts of immune cell infiltration in the disease but not in the controls so the real number of cells per spot is quite different between the conditions. Would you recommend running cell2location on the merged object with all samples and one n_cells_per_location parameter(Question then would be to use the high value from the condition or the low from the controls) or split the objects based on conditions and run them seperatly?
Thank you so much for your support!

**Comment by Zhongzheng99** [comment]
I'm also curious about this issue.
